causalml/propensity.py

```diff
@@ -66,7 +82,6 @@ def fit_predict(self, X, y):
         """
         self.fit(X, y)
         propensity_scores = self.predict(X)
-        logger.info("AUC score: {:.6f}".format(auc(y, propensity_scores)))
```

Comment:
question: why is this removed?

Reply:
It'd be outside of the function, so users can decide whether to check the AUC score or not.

Reply:
makes sense, thanks!